### PR TITLE
Document when to choose different types of local environments

### PR DESCRIPTION
**Pull request by deviantintegral**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +66,23 @@ Goals
...
 * No provisioning whatsoever; treat boxes as "fork and forget" for new projects.
 
+## Choosing between xAMP, Vagrant, and Docker
+
+Deciding what local development environment to choose for you and your team
+can be tricky. Here are three options, ordered in terms of complexity:
+
+1. Is your team entirely new to PHP and web development in general? Consider
+   using something like [MAMP](https://www.mamp.info) instead of Vagrant or
+   Docker.
```
Comment (q0rban) [inline README.md]: Should we mention Windows options? Is WAMP still a thing?

Comment (deviantintegral) [reply README.md]: MAMP actually has a windows variant now (!) https://www.mamp.info/en/downloads/

I don't know anything current to actually recommend for Windows though. Open to anything we know is good.

Comment (q0rban) [reply README.md]: > MAMP actually has a windows variant now (!)

😲 

Disregard then!

File: README.md [hunk 2]
```diff
@@ -29,6 +30,7 @@ you are new to Vagrant, read on.
 - [Email Configuration](#email-configuration)
 - [Drush Alias Autoconfiguration](#drush-alias-autoconfiguration)
 - [Verifying basebox integrity](#verifying-basebox-integrity)
+  - [Verifying the download when adding the box](#verifying-the-download-when-adding-the-box)
```
Comment (q0rban) [inline README.md]: I'm guessing this is just something that was missed in a previous PR?

Comment (deviantintegral) [reply README.md]: Yes, indeed.

**Pull request merged**